### PR TITLE
[i18nIgnore] docs: fix a few links with inconsistent locale

### PR DESCRIPTION
**Pull request by HiDeoo**
#### Description

This PR updates `starlight-links-validator` to the version `0.11.0` which now validates links in `<LinkCard>` and `<LinkButton>` components.

Following this update, the PR also fixes a few links in such components that were using inconsistent locales.

### CODE REVIEW

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 1]
```diff
@@ -237,7 +237,7 @@ import { LinkCard } from '@astrojs/starlight/components';
 
 import { LinkButton } from '@astrojs/starlight/components';
 
-<LinkButton href="/getting-started/">Get started</LinkButton>
+<LinkButton href="/zh-cn/getting-started/">Get started</LinkButton>
 <LinkButton href="https://docs.astro.build" variant="secondary" icon="external">
 	Related: Astro
 </LinkButton>
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/guides/components.mdx:243]: @liruifengv Looks like the texts ("Get started" & "Related: Astro") of the `<LinkButton>` example were forgotten during the translation, would you be up to fix them? You can do that in this PR or just suggest the changes in this PR if you prefer.

Comment (liruifengv) [reply docs/src/content/docs/zh-cn/guides/components.mdx:243]: Sure!

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 2]
```diff
@@ -237,7 +237,7 @@ import { LinkCard } from '@astrojs/starlight/components';
 
 import { LinkButton } from '@astrojs/starlight/components';
 
-<LinkButton href="/getting-started/">Get started</LinkButton>
+<LinkButton href="/zh-cn/getting-started/">Get started</LinkButton>
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:240]:
```suggestion
<LinkButton href="/zh-cn/getting-started/">开始使用</LinkButton>
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 3]
```diff
@@ -247,7 +247,7 @@ import { LinkButton } from '@astrojs/starlight/components';
...
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:250]:
```suggestion
<LinkButton href="/zh-cn/getting-started/">开始使用</LinkButton>
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 4]
```diff
@@ -247,7 +247,7 @@ import { LinkButton } from '@astrojs/starlight/components';
...
 <LinkButton href="https://docs.astro.build" variant="secondary" icon="external">
 	Related: Astro
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:252]:
```suggestion
	相关内容：Astro
```

File: docs/src/content/docs/zh-cn/guides/components.mdx [hunk 5]
```diff
@@ -237,7 +237,7 @@ import { LinkCard } from '@astrojs/starlight/components';
...
```
Comment (liruifengv) [inline docs/src/content/docs/zh-cn/guides/components.mdx:242]:
```suggestion
	相关内容：Astro
```

File: docs/src/content/docs/zh-cn/guides/components.mdx (hunk 1 above)
Comment (HiDeoo) [reply docs/src/content/docs/zh-cn/guides/components.mdx]: Amazing, thanks a lot, super appreciated 🙌

**Review by delucis — APPROVED** [review]
Thank you @HiDeoo and @liruifengv! LGTM 🚀 

Cool that the plugin can check these now. I'll have to take a look to see how you did it.

**Comment by HiDeoo** [comment]
> I'll have to take a look to see how you did it.

This was a fairly small change as in MDX, this is pretty much the same as an `<a>` tag. The next step will be supporting custom user components but taking the time to figure out the best way to let users define their own components/props.

**Pull request merged**